pkgs/development/python-modules/jupyterthemes/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, matplotlib
+, jupyter-core
+, lesscpy
+, notebook
+}:
+
+
+buildPythonPackage rec {
+  pname = "jupyterthemes";
+  version = "0.20.0";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Ko68DISyEquZufF1f8BYKj9Tkw06dbJJLZGnyLNqtB4=";
+  };
+
+  doCheck = true;
```

Comment:
```suggestion
```

This should be on by default for Python packages and does not need to be explicitly specified